penguin/web/middleware/auth.py

```
@@ -120,30 +131,34 @@ async def dispatch(self, request: Request, call_next):
+                    "detail": {
+                        "error": {
+                            "code": "AUTHENTICATION_FAILED",
+                            "message": str(e),
+                            "recoverable": False,
+                            "suggested_action": "check_credentials",
+                        }
                     }
                 },
                 headers={"WWW-Authenticate": "Bearer"},
             )
         except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)